KAS-4647: Enable new mu-auth for testing

sergiofenoll opened the pull request:
https://kanselarij.atlassian.net/browse/KAS-4647

sergiofenoll committed:
feat: enable new mu-auth for testing

sergiofenoll commented:
Jenkins builds:

- https://jenkins.kanselarij.org/job/kaleidos/job/custom_frontend_backend_combination/364/

sergiofenoll committed:
fix: mount new mu-auth config as well

sergiofenoll committed:
fix: use a feature branch of sparql-parser

This should contain a fix for ASK queries, making it possible to put
subcases on agendas again.

ValenberghsSven committed:
set image to latest after fixes

ValenberghsSven committed:
Merge branch 'development' of github.com:kanselarij-vlaanderen/app-kaleidos into feature/KAS-4647

ValenberghsSven committed:
Merge branch 'development' of github.com:kanselarij-vlaanderen/app-kaleidos into feature/KAS-4647

ValenberghsSven committed:
bump delta-notifier

ValenberghsSven committed:
Merge branch 'development' of github.com:kanselarij-vlaanderen/app-kaleidos into feature/KAS-4647

ValenberghsSven committed:
set a specific image for sparql-parser

ValenberghsSven commented:
Current issues:
some ECONNRESETS where happening. Perhaps due to yggdrasil bug locking up the system?
mocklogin sessions are not always created or deleted or valid.

ValenberghsSven committed:
Merge branch 'development' of github.com:kanselarij-vlaanderen/app-kaleidos into feature/KAS-4647

ValenberghsSven committed:
bump sparql-parser to most recent tag

ValenberghsSven committed:
add types to ovrb mu-auth config, fix type, add comments

ValenberghsSven committed:
bump sparq-parser to v0.0.4

ValenberghsSven committed:
log more resource

ValenberghsSven committed:
log more cache

ValenberghsSven committed:
Merge branch 'feature/KAS-4647' of github.com:kanselarij-vlaanderen/app-kaleidos into feature/KAS-4647

ValenberghsSven committed:
Merge branch 'development' of github.com:kanselarij-vlaanderen/app-kaleidos into feature/KAS-4647

ValenberghsSven committed:
bump sparql-parser

ValenberghsSven committed:
Merge branch 'development' of github.com:kanselarij-vlaanderen/app-kaleidos into feature/KAS-4647

ValenberghsSven committed:
Merge branch 'development' of github.com:kanselarij-vlaanderen/app-kaleidos into feature/KAS-4647

ValenberghsSven committed:
fix typo in config

ValenberghsSven committed:
Merge branch 'development' of github.com:kanselarij-vlaanderen/app-kaleidos into feature/KAS-4647

ValenberghsSven committed:
Merge branch 'development' of github.com:kanselarij-vlaanderen/app-kaleidos into feature/KAS-4647

ValenberghsSven commented:
merged the config changes to DEV already for easier deploy for testing

ValenberghsSven committed:
Merge branch 'development' of github.com:kanselarij-vlaanderen/app-kaleidos into feature/KAS-4647

ValenberghsSven committed:
bump sparql-parser

ValenberghsSven committed:
Merge branch 'development' of github.com:kanselarij-vlaanderen/app-kaleidos into feature/KAS-4647

ValenberghsSven committed:
restore commented error handling

ValenberghsSven committed:
revert back to 0.0.5

ValenberghsSven committed:
Merge branch 'development' of github.com:kanselarij-vlaanderen/app-kaleidos into feature/KAS-4647

ValenberghsSven committed:
parser 0.0.6 again

madnificent committed:
Switch to different mu-cl-resources branch

Hanldes async lockless type table differently

Co-author: @erikap

madnificent committed:
Increase max body size for case-documents-sync-service

Deltas are larger than what this accepts but it should be able to cope
with a larger size.  Increasing.

Co-author: @erikap

madnificent committed:
Fixate versions and remove unnecessary code

sergiofenoll committed:
Merge branch 'development' into feature/KAS-4647

sergiofenoll committed:
chore: add new auth rules to config

sergiofenoll committed:
Merge branch 'development' into feature/KAS-4647

sergiofenoll committed:
chore: remove duplicate graph and add internal-review

ValenberghsSven commented:
`submissions-write` still contains minister-roles. I think they should be read-only now.

ValenberghsSven committed:
Merge branch 'development' of github.com:kanselarij-vlaanderen/app-kaleidos into feature/KAS-4647

tdn committed:
Merge branch 'acceptance' of github.com:kanselarij-vlaanderen/app-kaleidos into feature/KAS-4647

tdn committed:
Use frontend with specific fixes for new mu-auth

ValenberghsSven committed:
fix typo in config

sergiofenoll committed:
chore: bump sparql-parser

ValenberghsSven commented on docker-compose.yml:
```diff
@@ -166,14 +165,16 @@ services:
       - "logging=true"
   agenda-approve:
-    image: kanselarij/agenda-approve-service:5.11.2
+    image: kanselarij/agenda-approve-service:feature-wait-on-busy-timeouts
```
we probably don't want this. this disables concurrency check on agenda approving which would result in wrong VR numbers

ValenberghsSven replied:
reverted back to tag and lower timeout in development docker file.

ValenberghsSven committed:
restore agenda-approve service tagged image

ValenberghsSven committed:
lower busy timeout for local development

ValenberghsSven commented on docker-compose.yml:
```diff
@@ -238,9 +239,11 @@ services:
       - "logging=true"
   case-documents-sync:
-    image: kanselarij/case-documents-sync-service:1.4.0
+    image: kanselarij/case-documents-sync-service:feature-max-body-size
     logging: *default-logging
     restart: always
+    environment:
+      MAX_BODY_SIZE: "500kb"
     labels:
```
MAX_BODY_SIZE is also declared in ENV properties in the PR `feature-max-body-size`
Do we need both?

ValenberghsSven committed:
add documentNamingJob type to mu-auth config

ValenberghsSven committed:
Merge branch 'development' of github.com:kanselarij-vlaanderen/app-kaleidos into feature/KAS-4647

tdn committed:
Merge branch 'acceptance' of github.com:kanselarij-vlaanderen/app-kaleidos into feature/KAS-4647

tdn committed:
chore: bump decision-report-generation to v1.0.1

tdn committed:
Merge branch 'acceptance' of github.com:kanselarij-vlaanderen/app-kaleidos into feature/KAS-4647

tdn committed:
bump images for sparql-parser

tdn reviewed (approved):
let's go!

tdn commented:
Accidentally merged this into development instead of acceptance, but pulled the changes from the branch directly afterward
